pkg/common/common.go

```diff
@@ -142,7 +142,7 @@ func ConstructLabelMap(appName, componentName string) map[string]string {
 	labels[constant.ComponentLabelKey] = componentName
 	labels[constant.ComponentLabelShortKey] = componentName
 	labels[constant.SelectorKey] = appName
-	labels[constant.FluentbitLabelKey] = appName
+	labels[constant.RsysLabelKey] = appName
```

Comment:
I suggest to use `csi.Name` as appName everywhere. In the most places current implementation is not related to `chart_release.Name`. For example - https://github.com/dell/csi-baremetal-operator/blob/935c886c152cd5633c3078d01f85061a08b3d20a/pkg/controller.go#L65